install.sh

```
@@ -335,8 +351,110 @@
+            if docker ps | grep -q "birdnet-go"; then
+                container_running=true
+                debug_output="${debug_output}Container with birdnet name running. "
+            fi
+        fi
+    fi
+    
+    # Debug output - uncomment to debug installation check
+    # print_message "DEBUG: $debug_output Service: $service_exists, Image: $image_exists, Container: $container_exists, Running: $container_running" "$YELLOW"
+    
+    # Check if Docker components exist (image or containers)
+    local docker_components_exist=$([ "$image_exists" = true ] || [ "$container_exists" = true ] || [ "$container_running" = true ] && echo true || echo false)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)